vmdb/app/models/miq_provision_openstack/cloning.rb

```diff
@@ -1,7 +1,7 @@
 module MiqProvisionOpenstack::Cloning
   def do_clone_task_check(clone_task_ref)
     source.with_provider_connection do |openstack|
-      instance = openstack.servers_for_accessible_tenants.detect { |s| s.id == clone_task_ref }
+      instance = openstack.handled_list(:servers).detect { |s| s.id == clone_task_ref }
```

Comment:
@blomquisg @Fryguy something tells me this is wrong, it should use get, not list right? otherwise cloning will be very ineffective

I think we will need servers_for_accessible_tenants alternative with get, so it fires up one get API request per each tenant